src/main/openapi/tenant-svc-openapi-internal.yaml

```diff
@@ -5,45 +5,131 @@ info:
-        summary: Create Customer
-        operationId: createCustomer
+          - TenantInternal
+        summary: Post new Tenant map
+        operationId: createTenantMap
         requestBody:
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Customer'
+            content:
+              application/json:
+                schema:
+                  $ref: '#/components/schemas/RequestTenantMap'
         responses:
-          '200':
-            description: OK
+          "201":
+            description: Created
+          "400":
+            description: Created
+            content:
+              application/json:
+                schema:
+                  $ref: '#/components/schemas/ErrorResponse'
+  /internal/tenant/{id}:
+      put:
+        tags:
+          - TenantInternal
+        summary: Update Tenant map by GUID
+        operationId: updateTenantMapById
+        parameters:
+        - name: id
+          in: path
+          required: true
+          schema:
+            type: string
+        requestBody:
             content:
-              'application/json':
+              application/json:
                 schema:
-                  $ref: '#/components/schemas/CustomerFullData'
+                  $ref: '#/components/schemas/RequestTenantMap'
+        responses:
+          "200":
+            description: OK
+          "404":
+            description: Not found
+
 components:
   schemas:
-    Customer:
+    RequestTenantMap:
+      required:
+        - InputTenantMap
+      type: object
+      properties:
+        inputTenantMap:
+          $ref: '#/components/schemas/InputTenantMap'
+    ResponseTenantMap:
+      required:
+        - TenantMap
+      type: object
+      properties:
+        tenantMap:
+          $ref: '#/components/schemas/TenantMap'
+    ResponseTenantMaps:
+      required:
+        - TenantMap
+      type: object
+      properties:
+        tenantMaps:
+          type: array
+          items:
+            $ref: '#/components/schemas/TenantMap'
+    InputTenantMap:
+      required:
+        - orgId
       type: object
       properties:
-        firstName:
+        orgId:
           type: string
-          description: First name of the customer
-        lastName:
+          description: Id of the organization
+    TenantMap:
+      type: object
+      properties:
+        orgId:
           type: string
-          description: Last name of the customer
-    CustomerFullData:
-      allOf:
-        - $ref: '#/components/schemas/Customer'
-        - type: object
-          properties:
-            customerId:
-              type: integer
-              description: The ID of the customer
-              format: int64
-      description: Full data of the customer.
\ No newline at end of file
+          description: Id of the organization
+        tenantId:
+          type: integer
+          description: Id of the tenant
+    ErrorResponse:
```

Comment:
Have you discussed about using this RFC? https://devonfw.com/docs/java/current/integration/rest_exception_handling.html